diagnostic_updater/include/diagnostic_updater/update_functions.h

```diff
@@ -116,23 +116,20 @@ namespace diagnostic_updater
        * \brief Constructs a FrequencyStatus class with the given parameters.
        */
 
-      FrequencyStatus(const FrequencyStatusParam &params) : 
-        DiagnosticTask("Frequency Status"), params_(params), 
+      FrequencyStatus(const FrequencyStatusParam &params, std::string name) :
+        DiagnosticTask(name), params_(params),
         times_(params_.window_size_), seq_nums_(params_.window_size_)
-    {
-      clear();
-    }
+      {
+        clear();
+      }
 
       /**
        * \brief Constructs a FrequencyStatus class with the given parameters.
+       *        Uses a default diagnostic task name of "Frequency Status".
        */
 
-      FrequencyStatus(const FrequencyStatusParam &params, const std::string name) :
-        DiagnosticTask(name), params_(params),
-        times_(params_.window_size_), seq_nums_(params_.window_size_)
-    {
-      clear();
-    }
+      FrequencyStatus(const FrequencyStatusParam &params) :
+        FrequencyStatus(params, "Frequency Status") {}
```

Comment:
While this is convenient, delegating constructors is a C++11 feature, and this package is still supporting Indigo which only supports C++03.